README.md

```diff
@@ -57,6 +62,8 @@ Cloud Datastore for your project.
 See the ``gcloud-java`` API [datastore documentation][datastore-api] to learn how to interact
```

Comment:
because this verbose documentation already exists in the service module (which is good) lets remove it from here
and like gcloud-node only 2 links (for javadoc and service official documentation).